### ISSUE TITLE
Augur.js won't connect to ipc w/ only options and callback params.

### ISSUE THREAD

**Issue by kevinday** [AugurProject/augur.js#83]
Here is a very simple reproducer:
```
var options = {
    http: "http://localhost:8545",
    ws: "http://127.0.0.1:8546",
    ipc: "/Users/k_day/Library/Ethereum/testnet/geth.ipc",
}

augur.connect(options, () => {
	console.log("connected")
});
```
ipc is getting null'd out at this line [https://github.com/AugurProject/augur.js/blob/aa2174473c8789ab8eb235af1c8d77a0148a83ef/src/modules/connect.js#L79](url) because the 3rd parameter (cb) isn't defined.

Also in the example I posted, the callback never gets called at all. Even if connect fails, i would expect this error to be passed back to the callback.

**Comment by kevinday** [comment]
There are a few places in code I've seen that are using connect in this way, so I assume it's usage we'd like to support, e.g, https://github.com/AugurProject/augur/blob/a82411554c63a530f0e4d11ee0f88bbfb326d781/src/services/augurjs.js#L34

**Comment by tinybike** [comment]
Fixed in 0feacbfabef3040894a862dc0a6ef0032f724d87
